build/templates/about_driver.inc.mako

```diff
@@ -0,0 +1,20 @@
+<%
+    import build.helper as helper
+
+    config        = template_parameters['metadata'].config
+    module_name   = config['module_name']
+    driver_name   = config['driver_name']
+%>\
+.. _about-section:
+
+About
+=====
+
+The **${module_name}** module provides a Python API for ${driver_name}. The code is maintained in the Open Source repository for `nimi-python <https://github.com/ni/nimi-python>`_.
+
+Support Policy
+--------------
+${module_name} supports all the Operating Systems supported by ${driver_name}.
+
+It follows `Python Software Foundation <https://devguide.python.org/#status-of-python-branches>`_ support policy for different versions.
+
```

Comment:
Here under support, I'd assume I could find links to Issues page and to NI forums. Do we have those in obvious locations?

Reply:
This isn't the support section. It's the (Operating System, Python) Support Policy section.

Issues links are in the status section.
https://github.com/ni/nimi-python/blob/caec980d4b68d536ffa2e1edd5683115f3bb38fa/generated/nidcpower/README.rst

![image](https://github.com/ni/nimi-python/assets/34140133/54f164ca-a06e-415e-914c-64eefae38a29)
Note: that's not the whole readme, just the top part.

Links to forums are in the Support section, further down the page. I think those locations are obvious enough.